templates/azure/terraform/example/hcl_properties.erb

```diff
@@ -24,7 +27,7 @@
         blocks.each do |block|
 -%>
 
-<%=       prop[0] -%> = {
+<%=       prop[0] -%> {
```

Comment:
Please make sure we include the equal sign (`=`) for tags (You can define a new helper function in `terraform/helpers.rb` similar to [this one for ansible](https://github.com/VSChina/magic-modules/blob/azure_backend/provider/azure/ansible/helpers.rb#L24-L26)). Here is the example in Terraform: https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_eventgrid_topic_test.go#L201.